src/main/php/web/frontend/AssetsFrom.class.php

```diff
@@ -25,12 +25,16 @@ class AssetsFrom extends FilesFrom {
     '*'        => ''
   ];
 
+  private $sources= [];
   private $preference;
 
-  /** @param io.Path|io.Folder|string $path */
-  public function __construct($path) {
-    parent::__construct($path);
+  /** @param io.Path|io.Folder|string|io.Path[]|io.Folder[]|string[] $sources */
+  public function __construct($sources) {
     $this->preferring(self::PREFERENCE);
+    foreach (is_array($sources) ? $sources : [$sources] as $source) {
+      $this->sources[]= $source instanceof Path ? $source : new Path($source);
+    }
+    parent::__construct($this->sources[0] ?? '.');
```

Comment:
This unnecessarily fills the *FilesFrom* class' `$path` member, which we do not use at all. See https://github.com/xp-forge/web/pull/102 on how we will be able to *use* the functionality of streaming static content instead of having to *inherit* from this base class.

Reply:
However, this would bump the version dependency to the feature level release for `xp-forge/web`, and at the same time would drop support for `^2.9`, which would be a BC break.